ipaserver/plugins/user.py

```diff
@@ -174,6 +174,27 @@ def check_last_member(user, protected_group_name=u'admins'):
         raise errors.LastMemberError(key=user, label=_(u'group'),
             container=protected_group_name)
 
+
+def is_in_local_idrange(command, uidnumber, options):
+    # Guaranteed, it will be autogenerated by sidgen
+    if uidnumber == -1:
+        return True
+
+    result = command.api.Command.idrange_find(
+        version=options['version'],
+        iparangetype='ipa-local',
+    )
```

Comment:
I wonder if someone would create so many ranges that some results would be truncated. Probably not but it wouldn't hurt to check.